ga4gh/cli.py

```diff
@@ -406,7 +418,7 @@ def addStartArgument(parser):
 
 def addEndArgument(parser):
     parser.add_argument(
-        "--end", "-e", default=1, type=int,
+        "--end", "-e", default=60000, type=int,
```

Comment:
Why 60000? I think this should be a very large value (i.e., the maximum possible value for the corresponding value in the Avro schema), so that we are guaranteed that this is always larger than the maximum in our data.